PokeAlarm/Events/RaidEvent.py

```diff
@@ -103,6 +106,10 @@ def generate_dts(self, locale, timezone, units):
             'weather': weather_name,
             'weather_or_empty': Unknown.or_empty(weather_name),
             'weather_emoji': get_weather_emoji(self.weather_id),
+            'boosted_weather_id': boosted_weather,
+            'boosted_weather': boosted_weather_name,
+            'boosted_weather_or_empty': Unknown.or_empty(boosted_weather_name),
+            'boosted_weather_emoji': get_weather_emoji(boosted_weather),
```

Comment:
`<boosted_weather_emoji>` does not resolve. 

![](https://cdn.discordapp.com/attachments/398270671794208789/405069643208851458/unknown.png)